pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -41,6 +41,13 @@ const (
 	// e.g 1m30s
 	BundleUnpackTimeoutAnnotationKey = "operatorframework.io/bundle-unpack-timeout"
 	BundleUnpackPodLabel             = "job-name"
+
+	// BundleUnpackRetryMinimumIntervalAnnotationKey sets a minimum interval to wait before
+	// attempting to recreate a failed unpack job for a bundle.
+	BundleUnpackRetryMinimumIntervalAnnotationKey = "operatorframework.io/bundle-unpack-min-retry-interval"
```

Comment:
Will you have a follow-up PR to document how to use this field?

Reply:
I'll follow up with https://github.com/operator-framework/olm-docs/issues/313 once the PR is merged